diff --git a/guidata/qt/QtGui.py b/guidata/qt/QtGui.py
index 2ac4bb5..75de219 100644
--- a/guidata/qt/QtGui.py
+++ b/guidata/qt/QtGui.py
@@ -14,7 +14,10 @@ if os.environ['QT_API'] == 'pyqt5':
     from PyQt5.QtGui import *                                 # analysis:ignore
     from PyQt5.QtWidgets import *                             # analysis:ignore
 elif os.environ['QT_API'] == 'pyqt':
-    from PyQt4.Qt import QKeySequence, QTextCursor            # analysis:ignore
+    try:
+        from PyQt4.Qt import QKeySequence, QTextCursor        # analysis:ignore
+    except ImportError:
+        from PyQt4.QtGui import QKeySequence, QTextCursor     # analysis:ignore
     from PyQt4.QtGui import *                                 # analysis:ignore
 else:
     from PySide.QtGui import *                                # analysis:ignore
